examples/next-modal-route/pages/index.js

```diff
@@ -0,0 +1,51 @@
+import React, { useState, useRef, useEffect } from "react";
+import Link from "next/link";
+import Modal from "../components/Modal";
+
+const App = () => {
+  const [modal, setModal] = useState(false);
+
+  const _modal = useRef(null);
+
+  const handleClick = () => {
+    setModal(true);
+    window.history.pushState("", "", "/test");
```

Comment:
This breaks Next.js / back button behavior, it should use the Next.js router instead.